tools/crdb-physical-replication.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+
+# Intended to automate https://docs.google.com/document/d/1vTX6N72J0yL2dDVdyJg063gC761QdZcMQ3ADthFYi8g/edit
+
+bold=$(tput bold)
+normal=$(tput sgr0)
+
+echo "${bold}CockroachDB Physical Replication setup${normal}"
+echo ""
+echo "This script will assist you in running a demo of Physical Replication on your local machine"
+echo ""
+
+echo "Checking prerequisites..."
+versionstr=$(cockroach version 2>&1)
+
+if [[ $versionstr == *"command not found"* ]]; then
```

Comment:
This is not a reliable way to determine whether crdb is installed or not.
Try this instead: `if ! cockroach version >dev/null 2>&1; then ...`

Reply:
Or alternatively, since you need the output:
```sh
if ! versionstr=$(cockroach version 2>/dev/null); then ...
```

Reply:
Also let's use `cockroach version --build-tag`.

Reply:
Good suggestions, done.